helpers/dbload.ts

```diff
@@ -96,6 +99,34 @@ UserLesson.belongsTo(User, {
 
 sequelize.sync({ alter: !!process.env.ALTER_DB })
 
+/* istanbul ignore next */
+if (process.env.NODE_ENV === 'development') {
+  //add data after fresh install
+  Lesson.findAll({
+    limit: 1
+  }).then((lessons: Array<LesssonTypescriptTypes>) => {
+    if (!lessons[0]) {
+      Lesson.bulkCreate(lessonData)
+      console.log('Successfully inserted lesson data')
```

Comment:
instead of console.log, we have an internal logger we use:

https://github.com/garageScript/c0d3-app/blob/master/helpers/middleware/logger.ts